test/misc.jl

```diff
@@ -20,6 +20,8 @@ function test_labels_codes(mg::MetaGraph)
         for label_0 in inneighbor_labels(mg, label_1)
             @test has_edge(mg, code_for(mg, label_0), code_for(mg, label_1))
         end
+        @test collect(all_neighbor_labels(mg, label_1)) ==
+            union(outneighbor_labels(mg, label_1), inneighbor_labels(mg, label_1))

```

Comment:
the union does not guarantee the same order, perhaps sorting first would be more robust

Reply:
Using sets could work too

Reply:
Thanks for the note. I have changed it so that sets are used.